processing/src/main/java/io/druid/query/DruidMetrics.java

```diff
@@ -59,6 +61,7 @@ public static int findNumComplexAggs(List<AggregatorFactory> aggs)
   public static <T> ServiceMetricEvent.Builder makePartialQueryTimeMetric(Query<T> query)
   {
     return new ServiceMetricEvent.Builder()
+        .setDimension(VERSION, Strings.nullToEmpty(DruidMetrics.class.getPackage().getImplementationVersion()))
```

Comment:
That method does not. I can refactor `makePartialQUeryTImeMetric` to omit setting that dimension if the value is null, otherwise I have to do this.